.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Does not belong in the repo, would be worth editing the gitignore.

Reply:
Well, if we edit the README to expect a venv folder in the workspace root, and edit the `pythonPath` setting here to be `${workspaceFolder}/venv/bin/python`, it's okay (maybe even better) for it to be okay in the repo.

Reply:
As somebody working primarily through conda on windows, I would worry about conflicts between such a setting and my usual workflow.

Reply:
> Does not belong in the repo, would be worth editing the gitignore.

Whops, this was mistakenly commited. Sorry!